alcs-frontend/src/app/features/application/decision/decision-v2/decision-documents/decision-documents.component.html

```diff
@@ -24,7 +24,12 @@ <h3>Documents</h3>
   <ng-container matColumnDef="fileName">
     <th mat-header-cell *matHeaderCellDef mat-sort-header sortActionDescription="Sort by name">Document Name</th>
     <td mat-cell *matCellDef="let element">
-      <a (click)="openFile(element.uuid, element.fileName)">{{ element.fileName }}</a>
+      <a
+        (click)="openFile(element.uuid, element.fileName)" 
+        [matTooltip]="element.fileName.length > fileNameTruncLen ? element.fileName: ''"
```

Comment:
Isn't this redundant? If the tooltip will be disabled, it doesn't really matter what the value is. I'd say just set this to always be `element.fileName` for simplicity.